src/java.base/share/classes/java/text/CompactNumberFormat.java

```diff
@@ -384,8 +384,8 @@ public final class CompactNumberFormat extends NumberFormat {
      * @param decimalPattern a decimal pattern for general number formatting
      * @param symbols the set of symbols to be used
      * @param compactPatterns an array of
-     *        <a href = "CompactNumberFormat.html#compact_number_patterns">
-     *        compact number patterns</a>
+     *        {@linkplain CompactNumberFormat##compact_number_patterns
```

Comment:
Can eliminate `CompactNumberFormat` here.

Reply:
Ah good point, it's the same file. Will make the change, thanks!